index.html

```diff
@@ -4409,6 +4438,122 @@ <h3 id="looping-AudioBufferSourceNode">
+              A property used to set the <code>EventHandler</code> (described
+              in <cite><a href=
+              "https://html.spec.whatwg.org/multipage/webappapis.html#eventhandler">
+              HTML</a></cite>[[!HTML]]) for the ended event that is dispatched
+              to <a><code>AudioBufferSourceNode</code></a> node types. When the
+              playback of the constant value is finished, an event of type
+              <code>Event</code> (described in <cite><a href=
+              "https://html.spec.whatwg.org/multipage/infrastructure.html#event">
+              HTML</a></cite> [[!HTML]]) will be dispatched to the event
+              handler.
+            </p>
+          </dd>
+          <dt>
+            void start(optional double when = 0)
+          </dt>
+          <dd>
+            <p>
+              Schedules the constant source to playback at an exact time. This
+              is defined exactly the same as for the <a href=
+              "#widl-AudioBufferSourceNode-start-void-double-when-double-offset-double-duration">
+              <code>start()</code></a> method for an
+              <a>AudioBufferSourceNode</a>, except that the the optional
+              <code>offset</code> and <code>duration</code> parameters are not
+              allowed.
+            </p>
+            <dl class="parameters">
+              <dt>
+                optional double when = 0
+              </dt>
+              <dd>
+                Defined the same as the <code>when</code> parameter of the
+                <code>start()</code> method for an
+                <a><code>AudioBufferSourceNode</code></a>.
+              </dd>
+            </dl>
+          </dd>
+          <dt>
+            void stop(optional double when = 0)
+          </dt>
+          <dd>
+            <p>
+              Schedules the source to stop playback at an exact time. This is
+              defined exactly the same as for the <a href=
+              "#widl-AudioBufferSourceNode-stop-void-double-when"><code>stop()</code></a>
+              method for an <a>AudioBufferSourceNode</a>
+            </p>
+            <dl class="parameters">
+              <dt>
+                optional double when = 0
+              </dt>
+              <dd>
+                Defined the same as the <code>when</code> parameter of the
+                <code>stop()</code> method for an
+                <a><code>AudioBufferSourceNode</code></a>
+              </dd>
+            </dl>
+          </dd>
+        </dl>
+        <section>
+          <h2>
+            ConstantSourceOptions
+          </h2>
+          <p>
+            This specifies options for constructing a
+            <a><code>ConstantSourceNode</code></a>. All members are optional;
+            if not specified, the normal defaults are used for constructing the
+            node.
+          </p>
+          <dl title="dictionary ConstantSourceOptions" class="idl">
+            <dt>
+              float sourceValue
```

Comment:
Do we really want to require this? The default is 1, so this wouldn't be required.